### PR TITLE
test: Test coverage for Note Command

### PR DESCRIPTION
**Pull request by dwangwk**
Cleaned up the NoteCommand quite substantially and added test cases that should be sufficient for v1.2.
80% Test Coverage for NoteCommand
100% Test Coverage for NoteCommandParser

### CODE REVIEW

**Review by yeoshuheng — APPROVED** [review]
ltgm, was wondering it if its worth limiting the note size, a potential issue i think could be the user adding a very long note, resulting in a failure to render in the UI

**Comment by dwangwk** [comment]
> ltgm, was wondering it if its worth limiting the note size, a potential issue i think could be the user adding a very long note, resulting in a failure to render in the UI

thanks for the input! i do believe this is a valid concern, perhaps we can implement this check in v1.3!

**Review by yespiggy — APPROVED** [review]
lgtm!

File: src/main/java/seedu/address/logic/commands/NoteCommand.java [hunk 1]
```diff
@@ -46,18 +35,18 @@ public class NoteCommand extends Command {
     public static final String MESSAGE_DUPLICATE_STARTUP = "This startup already exists in the address book.";
```
Comment (yespiggy) [inline src/main/java/seedu/address/logic/commands/NoteCommand.java:35]: This String seems not applicable in NoteCommand.

Comment (yespiggy) [inline src/main/java/seedu/address/logic/commands/NoteCommand.java:35]: This String `MESSAGE_DUPLICATE_STARTUP` seems not applicable in NoteCommand?

Comment (yespiggy) [inline src/main/java/seedu/address/logic/commands/NoteCommand.java:35]: The String `MESSAGE_NOT_EDITED` is never used.

**Review by yespiggy — COMMENTED** [review]
Just noticed two minor issues

**Pull request merged**